lib/utils/format.js

```diff
@@ -0,0 +1,14 @@
+export default (function() {
+  String.prototype.format = function () {
```

Comment:
It doesn't matter to me. The extension is easier to use, but I know it's frowned upon. Doesn't matter to me either way.